server/src/services/getAccounts.ts

```diff
@@ -1,12 +1,13 @@
 import { getFormattedUtcDatetime } from '../../../shared/helpers/datetime.js';
-import { respondWithServerError } from '../../../shared/helpers/errors.js';
 import { type AccountStatus } from '../../../shared/types';
+import { respondWithServerError } from '../helpers/errors.js';
 import { TxActions } from '../models/TxActions.js';
 import { TxTasks } from '../models/TxTasks.js';
 
 export const getAccounts = async (request: any, response: any) => {
-  const accountIds = request.body.accountId;
-  console.log('getAccounts accountIds', accountIds);
+  const { body } = request;
+  const accountIds = body.accountIds;
```

Comment:
Maybe you should do `const { accountIds } = body` to be consistent?

Reply:
I agree. I'll make that change.

Reply:
I think it will be possible for us to add a linter rule about this too. I'll add these ideas to a ticket we can do in the future to improve our linting.